Sources/Swift/Integrations/SessionReplay/SentryVideoInfo.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+
+@objcMembers
+class SentryVideoInfo: NSObject {
```

Comment:
This could be a `struct` if `SentrySessionReplay` were written in Swift, same for some of the other things in Swift/Integrations/SessionReplay